roles/common/postgres/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Install postgresql
+  apt:
+    name: postgresql
+    state: present
+    update_cache: yes
+
+- name: Install psycopg2
+  pip:
+    name: psycopg2
+    version: 2.7.3.2
+    state: present
+
+- name: Create postgres role for custom user
+  become: true
+  become_user: postgres
```

Comment:
The required `postgres` user is not created within the `postgres` role (so the role cannot be used on its own).

Reply:
I don't believe `postgres` is a user that needs creating. I think it's set up automatically somewhere so will add a comment to clarify this.